chat-widget/src/components/webchatcontainerstateful/common/mockchatsdk.ts

```diff
@@ -58,4 +58,8 @@ export class MockChatSDK {
             reconnectId: "123"
         };
     }
+
+    public getVoiceVideoCalling() {
```

Comment:
is this just to make existent test pass or is pending to add test for this new scenario?

Reply:
Yes this additional method in the mock allowed for the existing VRT test case to pass.

A VSO test case will be added as part of the script layer uptake PR that will come after this widget layer PR merges in.